test/tab-components/frameTest.js

```diff
@@ -278,6 +279,20 @@ describe('frame tests', function () {
         .url(url)
         .waitForVisible('#viewerContainer')
     })
+
+    it('loads HTML properly despite having .pdf extension', function * () {
```

Comment:
thanks for adding a test! unfortunately our automated webdriver test environment is totally broken at the moment, so instead i ran the custom server and manually loaded the .pdf HTML file. confirmed that it is broken in latest release and fixed with this PR.